src/pages/Homepage.tsx

```diff
@@ -0,0 +1,18 @@
+import { PageSection } from "../components/PageSection/PageSection";
+import { BentoMenu } from "../components/BentoMenu/BentoMenu";
+import { TextField } from "../components/TextField/TextField";
+import { CTO } from "../components/CTO/CTO";
+import { Calendar } from "@/components/ui/calendar";
+
+export const Homepage = () => {
+	return (
+		<>
+			<PageSection>
+				<Calendar />
```

Comment:
Ani nevím, jestli to jde používat jen takhle, asi pak budeš muset specifikovat nějaký props. V dokumentaci mají příklad tohle, ale najdi si to pak sama:
```
<Calendar
  mode="single"
  selected={date}
  onSelect={setDate}
/>
```